main.go

```diff
@@ -110,3 +127,32 @@ func main() {
 
 	app.Logger.Fatal(app.Start(util.BindAddress))
 }
+
+func LookupEnvOrString(key string, defaultVal string) string {
+	if val, ok := os.LookupEnv(key); ok {
+		return val
+	}
+	return defaultVal
+}
+
+func LookupEnvOrBool(key string, defaultVal bool) bool {
+	if val, ok := os.LookupEnv(key); ok {
+		v, err := strconv.ParseBool(val)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "LookupEnvOrInt[%s]: %v\n", key, err)
+		}
+		return v
+	}
+	return defaultVal
+}
+
+func LookupEnvOrInt(key string, defaultVal int) int {
+	if val, ok := os.LookupEnv(key); ok {
+		v, err := strconv.Atoi(val)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "LookupEnvOrInt[%s]: %v\n", key, err)
+		}
+		return v
+	}
+	return defaultVal
+}
```

Comment:
Can you move these functions to `util/util.go`?

Reply:
Yes. I can. But I think it would be better if you reject this PR, and I rewrote it using the viper library. https://github.com/spf13/viper
What do you think?

Reply:
hmm, I don't know. To me, it looks like using viper is an overkill solution for this purpose. I would prefer simple ones.

Reply:
No problem. Lets stick to the original plan. I'll move the functions to util.

Reply:
Changes made and smoke tested. If I werent so lazy, I would write some real tests.